sygnal/webpushpushkin.py

```diff
@@ -18,22 +18,32 @@
 from base64 import urlsafe_b64encode
 from hashlib import blake2s
 from io import BytesIO
-from typing import List, Optional, Pattern
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Pattern
 from urllib.parse import urlparse
 
+import py_vapid
 from prometheus_client import Gauge, Histogram
 from py_vapid import Vapid, VapidException
```

Comment:
Nit: it would be nicer to import `py_vapid.CaseSensitiveDict` on line 26 of the RHS here, along with the other imports from that module.